src/assigner.py

```diff
@@ -0,0 +1,46 @@
+from settings import ID_DICT_LIST_ITEM, ASSIGN_TO_TEST_PLANS, ASSIGN_TO_FEATURE, TP_URL, ACCESS_TOKEN
+from copy import deepcopy
+import requests
+
+class Assigner():
+    def __init__(self, feature=None, test_plan=None, test_case=None):
+        self.feature = feature
+        self.test_plan = test_plan[0]
+        self.test_case = test_case
+        self.data = self.set_data()
+        self.request_url = self.set_request_url()
+
+    def set_data(self):
+        if(self.feature == None):
```

Comment:
If you're equalling to None in python you can use `not` 
```suggestion
        if not self.feature:
```